examples_utils/benchmarks/run_benchmarks.py

```diff
@@ -152,6 +153,47 @@ def proc_thread():
     return (output, err, exitcode)
 
 
+def install_patched_requirements(requirements_file: Union[str, Path], listener: TextIOWrapper):
```

Comment:
This is only in the case where a notebook calls `pip install -r requirements.txt` for an example or something right? In an env that already has examples_utils installed? 

i.e this will never be called when running traditional examples that dont re-install requirements?

If so, should this not be in notebook_utils? Just want to keep notebook-only things in one place.

Reply:
When I added requirements installation I added it so that any benchmark could ask to install requirements, this was because I figured there was nothing specific "about installing requirements for notebooks" rather than for a script. So the need to patch the requirements file arises when I try to use that to install the app requirements from the tool.

Reply:
I dont see a case for installing app requirements using this benchmarking module outside of notebooks, its never been a thing because by definition, examples utils only comes from installing pip requirements manually right?